lib/template/template.mustache

```diff
@@ -181,6 +181,44 @@
 						<h4 class="custom-font-subtitle"><strong>{{heading}}</strong></h4>
 						<br/>
 						{{/heading}}
+						
+
+
+						<!-- Seção de testes de novas funções -->
+						<!-- Função Tablela -->
+						<table class="table table-hover">
+							{{#tabela}}
+							<tr>
+								{{#title}}<th colspan="5" class="text-center">{{.}}</th>{{/title}}
+							<tr>
+								<tr>
+									{{#th}}
+										<th>{{Col1}}</th>
```

Comment:
The code only works when the table has five columns???

Reply:
Five columns is the maximum, I limited to five due to the mustache loop type, so that the mustache can loop the TD they have to have a name (variable), otherwise it can't loop ex: line01 , line02 ...

Reply:
Mustache has a basic logical for the empty fields, I'll training later.